docs/wip/demo_dapr.md

```diff
@@ -0,0 +1,295 @@
+
+    1. Disable permissive mode: 
+
+       ```console
+       $ kubectl patch meshconfig osm-mesh-config -n osm-system -p '{"spec":{"traffic":{"enablePermissiveTrafficPolicyMode":false}}}'  --type=merge
+       meshconfig.config.openservicemesh.io/osm-mesh-config patched
+       ```
+    
+    2. Verify the pythonapp documented [here](https://github.com/dapr/quickstarts/tree/master/hello-kubernetes#step-6---observe-messages) no longer causes the order ID to increment.
+
+    3. Create a service account for nodeapp and pythonapp: 
+
+       ```console
+       $ kubectl create sa nodeapp -n dapr-test
+       serviceaccount/nodeapp created
+       ```
+
+       ```console
+       $ kubectl create sa pythonapp -n dapr-test
+       serviceaccount/pythonapp created
+       ```
+  
+    4. Update the role binding on the cluster to contain the newly created service accounts:
+
+       ```bash
+       kubectl apply -f - <<EOF
+       ---
+       kind: RoleBinding
+       apiVersion: rbac.authorization.k8s.io/v1
+       metadata:
+         name: dapr-secret-reader
+         namespace: dapr-test
+       subjects:
+       - kind: ServiceAccount
+         name: default
+       - kind: ServiceAccount
+         name: nopdeapp
+       - kind: ServiceAccount
+         name: pythonapp
+       roleRef:
+         kind: Role
+         name: secret-reader
+         apiGroup: rbac.authorization.k8s.io
+       EOF
+       ```
+
+    5. Apply the following SMI access control policies:
+
+       Deploy SMI TrafficTarget
+       ```bash
+       kubectl apply -f - <<EOF
+       ---
+       kind: TrafficTarget
+       apiVersion: access.smi-spec.io/v1alpha3
+       metadata:
+         name: pythodapp-traffic-target
+         namespace: dapr-test
+       spec:
+         destination:
+           kind: ServiceAccount
+           name: nodeapp
+           namespace: dapr-test
+         rules:
+         - kind: HTTPRouteGroup
+           name: nodeapp-service-routes
+           matches:
+           - new-order
+         sources:
+         - kind: ServiceAccount
+           name: pythonapp
+           namespace: dapr-test
+       EOF
+       ```
+
+       Deploy HTTPRouteGroup policy
+       ```bash
+       kubectl apply -f - <<EOF
+       ---
+       apiVersion: specs.smi-spec.io/v1alpha4
+       kind: HTTPRouteGroup
+       metadata:
+         name: nodeapp-service-routes
+         namespace: dapr-test
+       spec:
+         matches:
+         - name: new-order
+       EOF
+       ```
+
+    6. Update the pod spec in both nodeapp ([node.yaml](https://github.com/dapr/quickstarts/blob/master/hello-kubernetes/deploy/node.yaml)) and pythonapp ([python.yaml](https://github.com/dapr/quickstarts/blob/master/hello-kubernetes/deploy/python.yaml)) to contain their respective service accounts. Delete and re-deploy the Dapr hello-kubernetes pods
+
+    7. Verify the Darp hello-kubernetes demo works as expected
```

Comment:
I'd suggest to add a log-snippet/screen of what to expect when it's working.

Reply:
The log snippet of this is provided the Dapr demo, I am not sure if it makes sense duplicating that here. I have added a link to the Dapr doc instead